samples/azure/custom-audit-transport/Core_9/CustomAuditTransport/AuditTransportBehavior.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading.Tasks;
+using CustomAuditTransport;
+using NServiceBus;
+using NServiceBus.Pipeline;
+
+public class AuditTransportBehavior :
+    Behavior<IAuditContext>
+{
+
+
+    public AuditTransportBehavior()
+    {
+
+    }
+
+    public override async Task Invoke(IAuditContext context, Func<Task> next)
+    {
+        //currently this fails as what is being sent is NServiceBus.Transport.OutgoingMessage - that's not something nsb recognises as a message/command/event
```

Comment:
Is there a reason why next isn't invoked?

Reply:
Yes, we don't want the audit message to be sent on ASB.  The behavior dispatches the audit message on ASQ and then terminates the pipeline.

Reply:
We are now using a terminating pipeline behaviour instead so I am resolving this comment.